desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.component.ts

```diff
@@ -57,7 +59,9 @@ export class FixOutliersModalComponent {
 
         this.projectDocument = await this.datastore.get('project');
         this.valuelist = await this.getValuelist(this.document, this.field);
+        this.inputType = await this.field.inputType;
```

Comment:
"await" is not needed here (it's only needed when waiting for a promise to resolve).

It's also not really necessary to store the input type in a new public property as it can simply be accessed via "this.field.inputType" (also in the html template via "field.inputType").